### ISSUE TITLE
masterLooterGuid not found (BB #853)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#853]
This issue was migrated from bitbucket.
**Original Reporter:** bbazarragchaa
**Original Date:** 01.05.2015 14:11:57 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/853

<hr>

I have following db error:

```
#!sql
2015-05-01 18:28:06 SQL: INSERT INTO groups(leaderGuid,mainTank,mainAssistant,lootMethod,looterGuid,lootThreshold,icon1,icon2,icon3,icon4,icon5,icon6,icon7,icon8,isRaid,difficulty,masterLooterGuid) VALUES('96','0','0','3','96','2','0','0','0','0','0','0','0','0','0','0','0')
2015-05-01 18:28:06 SQL ERROR: Unknown column 'masterLooterGuid' in 'field list'
```

Revision: ab3d3cb1540503e1eaedc0c143de25aa77e8dbcc

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 07.05.2015 14:24:58 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/853/#comment-17944156

<hr>

Disregard that, two in one day. On a roll!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 07.05.2015 14:33:19 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/853/#comment-17944378

<hr>

[DB/Characters] Add missing SQL. Fixes #853

→ <<cset 35521cf1bb28>>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** bbazarragchaa
**Original Date:** 08.05.2015 03:44:51 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/853/#comment-17957865

<hr>

Thank you for the fix, I applied all the SQL updates BTW.
